utils/generic-components.stanza

```diff
@@ -266,6 +176,41 @@ public defn look-up-ceramic-caps (attribute: String, filter-properties:Tuple<Key
                                         filter-properties]
   look-up-capacitors(attribute, params)
 
+; ========================================================
+; ==================== tantalum-cap =======================
+; ========================================================
+
+public defn tantalum-cap (user-params:Tuple<KeyValue>) -> Instantiable :
+  val params = remove-duplicate-keys $ [["anode" => "tantalum" "min-rated-voltage" => MIN-CAP-VOLTAGE]
+                                        smd-query-properties()
+                                        user-params]
+  to-jitx $ Capacitor(params)
+
+public defn tantalum-cap (cap:Double) -> Instantiable :
+  tantalum-cap(["capacitance" => cap])
+
+public defn tantalum-cap (cap:Double, tolerance:Double) -> Instantiable :
+  tantalum-cap(["capacitance" => cap "tolerance" => tolerance])
+
+public defn tantalum-caps (user-params:Tuple<KeyValue>, limit: Int) -> Tuple<Instantiable> :
```

Comment:
This function is interesting, it returns a tuple of valid components that satisfy your query. How would it be used in context, compared to using `lookup-tantalum-caps' and selecting one?

Reply:
pass - this gets captured in our docs